ui/redux/actions/comments.js

```diff
@@ -470,18 +470,10 @@ export function doCommentReact(commentId: string, type: string) {
  * @param {string} [environment] Optional environment for Stripe (test|live)
  * @returns {(function(Dispatch, GetState): Promise<undefined|void|*>)|*}
  */
-export function doCommentCreate(
-  comment: string = '',
-  claim_id: string = '',
-  parent_id?: string,
-  uri: string,
-  livestream?: boolean = false,
-  txid?: string,
-  payment_intent_id?: string,
-  environment?: string,
-  sticker: boolean
-) {
+export function doCommentCreate(uri: string, livestream: boolean, params: CommentSubmitParams) {
```

Comment:
The documentation of this function requires an update with the `CommentSubmitParams` change.
But removing it entirely is also fine since params are self-documenting.  The existing explanation is also just stating the obvious.